modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/DumpWGAExpectedTags.pm

```diff
@@ -0,0 +1,71 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::GeneTrees::DumpWGAExpectedTags
+
+=head1 DESCRIPTION
+
+A runnable to dump wga_expected tag and delete it from a database.
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::GeneTrees::DumpWGAExpectedTags;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Compara::RunnableDB::BaseRunnable');
+
+
+sub run {
+    my ($self) = shift @_;
+   
+    $self->dump_wga_expected();
+
+    $self->delete_wga_expected();
```

Comment:
Just a reminder of my comment in the other PR: these two functions are really small so you could move their code in here (less lines of code, easier to follow and read in one go).

Reply:
Yes, I remember your comment @JAlvarezJarreta. That's the only one that I initially didn't do anything about as I wanted to hear @carlacummins opinion ;)